egress/src/patchwork.rs

```diff
@@ -591,13 +614,20 @@ pub fn get_applicable_timeseries(
         });
         // have overlap
         if let Some(t) = overlap {
-            applicable_ts.push((f.tsid, t.from.unwrap(), t.to.unwrap_or(to)));
+            applicable_ts.push(Patch {
+                tsid: f.tsid,
+                from: t.from.unwrap(),
+                to: t.to.unwrap_or(to),
```

Comment:
I guess we use the users to date, so its not open or we have to use the current time if they didn't specify?

Reply:
Right now I think you are required to send from/to times in the request, so to can't be open